riotwatcher/_apis/BaseApi.py

```diff
@@ -46,3 +47,59 @@ def request(self, endpoint_name, method_name, region, url_ext, **kwargs):
                     response = mod
 
         return response
+
+    def request_static(self, version, locale, url_ext):
+        url = 'http://ddragon.leagueoflegends.com/cdn/{version}/data/{loc}/{ext}.json'.format(version=version,
+                                                                                              loc=locale, ext=url_ext)
+
+        response = None
+        early_ret_idx = None
+
+        if self._request_handlers is not None:
+            for idx, handler in enumerate(self._request_handlers, start=1):
+                response = handler.preview_static_request(version, locale, url)
+                early_ret_idx = idx
+                if response is not None:
+                    break
+
+        if response is None:
+            response = requests.get(url)
+
+        if self._request_handlers is not None:
+            for handler in self._request_handlers[early_ret_idx:None:-1]:
+                mod = handler.after_static_request(version, locale, url, response)
+                if mod is not None:
+                    response = mod
+
+        return response
+
+    def request_version(self, endpoint_name, method_name, region):
+        region = re.sub('\d', '', region)
+        url = 'https://ddragon.leagueoflegends.com/realms/{region}.json'.format(region=region)
+
+        response = None
+        early_ret_idx = None
+
+        if self._request_handlers is not None:
+            for idx, handler in enumerate(self._request_handlers, start=1):
+                response = handler.preview_request(
```

Comment:
I'll change it but I wonder if it would still be better as preview_request since preview_static_request takes a version(which we don't have) a locale(which we don't have) and the url. Should I maybe just need other handler method for the version request?